src/pages/iou/steps/MoneyRequstParticipantsPage/MoneyRequestParticipantsSplitSelector.js

```diff
@@ -178,7 +173,7 @@ class MoneyRequestParticipantsSplitSelector extends Component {
         if (isOptionInList) {
             newSelectedOptions = _.reject(this.props.participants, (selectedOption) => selectedOption.accountID === option.accountID);
         } else {
-            newSelectedOptions = [...this.props.participants, option];
+            newSelectedOptions = [...this.props.participants, {accountID: option.accountID, login: option.login, selected: true}];
```

Comment:
I keep the `login` property here because OptionsSelector hasn't been migrated to `accountID` and still using `login`.

_In Confirmation list page, we also use OptionsSelector, but the participants is mapped with OptionsListUtils.getParticipantsOptions that contains `login`_